packages/babel-preset-css-prop/src/index.js

```diff
@@ -13,17 +13,22 @@ export default (
   api,
   { pragma, sourceMap, autoLabel, labelFormat, instances, ...options } = {}
 ) => {
+  const isAutomaticRuntime = options.runtime === 'automatic'
+  console.log({ isAutomaticRuntime })
+  const jsxOptions = isAutomaticRuntime
+    ? { importSource: '@emotion/core' }
+    : { pragma: pragmaName, pragmaFrag: 'React.Fragment' }
   return {
     plugins: [
-      [
+      !isAutomaticRuntime && [
```

Comment:
We don't need this since the module the automatic runtime uses to resolve `jsx` is based on `importSource`.